### ISSUE TITLE
search pattern in issues

### ISSUE THREAD

**Issue by Sh3idan** [maxtepkeev/python-redmine#209]
As describe here: https://python-redmine.com/introduction.html#search

I do:
```
from redminelib import Redmine

redmine = Redmine("url")
issues = redmine.issue.search("foo")
```

And I have an exception : redminelib.exceptions.ResourceNotFoundError. If I do the same request via the webui, I have some results.

Can you help me ? :)

**Comment by maxtepkeev** [comment]
Search API is available starting from Redmine 3.3, what version of Redmine do you have ?

**Comment by Sh3idan** [comment]
Hum, that's why. Thanks for your answer and thank you for the lib.
